### ISSUE TITLE
sidetag: do not add builds which were not built in it

### ISSUE THREAD

**Issue by igor-raits** [fedora-infra/bodhi#3778]
With Rust, I'm tagging packages from rawhide into side tag and then performing build of actual packages. However, bodhi wants to create update with all those packages in it. I believe that we want to only suit builds which were built in that side tag.

As for non-rust case, you can tag older gcc build into the side tag, but you don't want to submit update for it.

**Comment by igor-raits** [comment]
@ryanlerch @bowlofeggs @sebwoj thoughts?

**Comment by bowlofeggs** [comment]
There might be a use case for people to tag things in to a side tag with the intent to have them shipped. But I agree that if people are tagging things in just to be part of the build root, they don't necessarily want them shipped. I wonder if the side tag should get a side tag build root as well, so that people have a way to express those two different intents.

**Comment by igor-raits** [comment]
I don't think we want to maintain separate -build for side tags... It is just overcomplication.

If you think this is useful, I can send PR.

**Comment by bowlofeggs** [comment]
I don't work on Bodhi anymore, so I'll defer to those who do as to whether it would be useful ☺

**Comment by cverna** [comment]
@ignatenkobrain Can you explain a little bit more why you would need to add a build that is in the rawhide tag (ie f32) in a side-tag ? If the build is in the rawhide tag it should already be available in the buildroot ?

Sorry if I am missing out something obvious but I prefer to have a clear understanding of the need :-)

**Comment by igor-raits** [comment]
yes, but the buildroot is for f31. so we tag crates from rawhide into f31 buildroot.

but this is very specific case, more generic one would be when you want to tag older gcc into your side tag in order to build your package. then you don't want to submit update for this older gcc.
